### PR TITLE
Feature/cvsb 16663

### PR DESCRIPTION
**Pull request by MartiUK**
Update dependencies
update packages for security vulnerabilities https://jira.dvsacloud.uk/browse/CVSB-16663

### CODE REVIEW

File: jest.config.int.js [hunk 1]
```diff
@@ -1,4 +0,0 @@
-const config = require('./jest.config');
```
Comment (JonLazariniDD) [inline jest.config.int.js:1]: Why did we remove that?
are the tests still running?

Comment (MartiUK) [reply jest.config.int.js:1]: It was an unused file breaking with the newest jest/tsjest

**Comment by MartiUK** [comment]
@JonLazariniDD I have aligned this repo with the others which is to use npm, not yarn.

File: jest.config.int.js (hunk 1 above)
Comment (JonLazariniDD) [reply jest.config.int.js:1]: thanks resolving

**Pull request merged**